### DISCUSSION TITLE
Anybody get this to work with Expo?

### DISCUSSION THREAD

**Discussion by adibkadir** [DylanVann/react-native-fast-image#930] [Q&A]
I get "Invariant Violation: requireNativeComponent: "FastImageView" was not found in the UIManager." out of the box.

**Comment by saitgurdag** [comment]
Did you find that, how can we use it with expo?

**Comment by eeshankeni** [comment]
You cannot use this with expo go. Instead, install the package and [generate a development client](https://docs.expo.dev/development/create-development-builds/)
